types/json.go

```diff
@@ -105,6 +105,16 @@ func (a JSONArray) String() string {
 	return o.String()
 }
 
+func (a *JSONArray) ValueString(i int) string {
+	arr := a.Storage().(*array.Binary)
+	switch {
+	case a.IsNull(i):
+		return "(null)"
+	default:
+		return fmt.Sprintf("\"%s\"", arr.Value(i))
```

Comment:
```suggestion
		return fmt.Sprintf("%q", arr.Value(i))
```
So that the value itself can be quoted as well?